### PR TITLE
Update instructions

### PR DESCRIPTION
**Pull request by dabria**
Updated the instructions in a couple spots to help less familiar engineers and corrected a part which got me stuck.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -150,6 +150,8 @@ autovacuum_analyze_scale_factor = 0.025
 
 The following steps are intended for Sensu Engineering use, they are shared here for transparency.
 
+**Check the pins in the #engineering channel in Slack for the IP of the SSH jump host.**
```
Comment (echlebek) [inline README.md]: There is now a DNS name associated with the IP address, it is `spdc.sensu.io`.

For now, people will still need to use the IP address for accessing the graphana dashboard, but you can already use this DNS name for connecting to the SSH server.

File: README.md [hunk 2]
```diff
@@ -246,23 +249,30 @@ ssh backend1
...
 
 sensuctl create -f check3.yml
 
 sensuctl create -f check4.yml
+
+...
 ```
 
-Use Grafana to observe system performance. Watch service logs for any
+Use Grafana to observe system performance. Grafana runs on port 300 of the SSH jump host. Watch service logs for any
```
Comment (echlebek) [inline README.md]: Port 3000, not 300

Comment (dabria) [reply README.md]: Whoops, good catch!

File: README.md (hunk 1 above)
Comment (dabria) [reply README.md]: Good to know, I'll add that as well!

**Pull request merged**